ext/node/polyfills/internal/util/inspect.mjs

```diff
@@ -302,1614 +124,6 @@ defineColorAlias("inverse", "swapColors");
-      point < 32 ||
-      (point > 126 && point < 160)
-    ) {
-      if (last === i) {
-        result += meta[point];
-      } else {
-        result += `${str.slice(last, i)}${meta[point]}`;
-      }
-      last = i + 1;
-    }
-  }
-
-  if (last !== lastIndex) {
-    result += str.slice(last);
-  }
-  return addQuotes(result, singleQuote);
-}
-
-function stylizeWithColor(str, styleType) {
-  const style = inspect.styles[styleType];
-  if (style !== undefined) {
-    const color = inspect.colors[style];
-    if (color !== undefined) {
-      return `\u001b[${color[0]}m${str}\u001b[${color[1]}m`;
-    }
-  }
-  return str;
-}
-
-function stylizeNoColor(str) {
-  return str;
-}
-
-// Note: using `formatValue` directly requires the indentation level to be
-// corrected by setting `ctx.indentationLvL += diff` and then to decrease the
-// value afterwards again.
-function formatValue(
-  ctx,
-  value,
-  recurseTimes,
-  typedArray,
-) {
-  // Primitive types cannot have properties.
-  if (
-    typeof value !== "object" &&
-    typeof value !== "function" &&
-    !isUndetectableObject(value)
-  ) {
-    return formatPrimitive(ctx.stylize, value, ctx);
-  }
-  if (value === null) {
-    return ctx.stylize("null", "null");
-  }
-
-  // Memorize the context for custom inspection on proxies.
-  const context = value;
-  // Always check for proxies to prevent side effects and to prevent triggering
-  // any proxy handlers.
-  // TODO(wafuwafu13): Set Proxy
-  const proxy = undefined;
-  // const proxy = getProxyDetails(value, !!ctx.showProxy);
-  // if (proxy !== undefined) {
-  //   if (ctx.showProxy) {
-  //     return formatProxy(ctx, proxy, recurseTimes);
-  //   }
-  //   value = proxy;
-  // }
-
-  // Provide a hook for user-specified inspect functions.
-  // Check that value is an object with an inspect function on it.
-  if (ctx.customInspect) {
-    const maybeCustom = value[customInspectSymbol];
-    if (
-      typeof maybeCustom === "function" &&
-      // Filter out the util module, its inspect function is special.
-      maybeCustom !== inspect &&
-      // Also filter out any prototype objects using the circular check.
-      !(value.constructor && value.constructor.prototype === value)
-    ) {
-      // This makes sure the recurseTimes are reported as before while using
-      // a counter internally.
-      const depth = ctx.depth === null ? null : ctx.depth - recurseTimes;
-      const isCrossContext = proxy !== undefined ||
-        !(context instanceof Object);
-      const ret = maybeCustom.call(
-        context,
-        depth,
-        getUserOptions(ctx, isCrossContext),
-      );
-      // If the custom inspection method returned `this`, don't go into
-      // infinite recursion.
-      if (ret !== context) {
-        if (typeof ret !== "string") {
-          return formatValue(ctx, ret, recurseTimes);
-        }
-        return ret.replace(/\n/g, `\n${" ".repeat(ctx.indentationLvl)}`);
-      }
-    }
-  }
```

Comment:
It would be better to keep the customInspect support for compatibility.

Reply:
The idea here is to align Deno.inspect and util.inspect into the single implementation as they have lot of duplicated algorithms.

I think we should move these (Node.js specific) custom inspect features into Deno.inspect implementation in later PRs.